src/procedures/__tests__/CreateSecurityToken.ts

```diff
@@ -142,9 +143,30 @@ describe('CreateSecurityToken', () => {
 
       // Real call
       const resolver = await target.prepareTransactions();
-      await resolver.run({} as TransactionReceiptWithDecodedLogs);
+      const receipt = {} as TransactionReceiptWithDecodedLogs;
+      await resolver.run(receipt);
+
+      // Verification for resolver result
       expect(resolver.result).toEqual(creationObject);
+      // Verification for fetch
+      expect(createStub.getCall(0).calledWithMatch(SecurityToken.generateId, {})).toEqual(true);
```

Comment:
Why is this check here if the next one is more specific?

Reply:
Good catch